store/blocks.jsonc

```diff
@@ -94,5 +94,10 @@
   },
   "store.not-found#search": {
     "blocks": ["rich-text#not-found"]
+  },
```

Comment:
@klzns you could use this PR to add a gif of this example on the documentation! It still lacks the image

Reply:
Great idea! Done: https://github.com/vtex-apps/disclosure-layout/pull/3